src/lightbox/Lightbox.js

```diff
@@ -76,6 +76,8 @@ export default function Lightbox(props: Props) {
         //   https://github.com/alwx/react-native-photo-view/issues/98
         // TODO: Figure out how to make it work.
         resizeMode="contain"
+        showsHorizontalScrollIndicator={false}
+        showsVerticalScrollIndicator={false}
```

Comment:
Lightbox: Remove scrollbars when panning and zooming on iOS.
    
    I don't think these really add anything.

Agreed.

From trying the current beta on Android, I think the "on iOS" means that currently we have scrollbars on iOS but not Android, and after this change we don't have them on either platform. Does that agree with what you have in mind? Would be good to make explicit -- otherwise it sounds like perhaps we still have them but only on Android.

Reply:
Right -- these are iOS-only props. I'll make a note.